bin/server.py

```diff
@@ -0,0 +1,130 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main module of cusdeb-helpik"""
+import json
+import logging
+
+import aiohttp_cors
+import requests
+from aiohttp import web
+from aiohttp.web_request import Request
+from aiohttp_cors import CorsViewMixin
+from bs4 import BeautifulSoup
+
+from bin import config
+
+logging.basicConfig(filename='cusdeb-helpik.log', level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+class Handler(web.View, CorsViewMixin):
+    """Class contains handlers and auxiliary methods for them"""
+    def __init__(self, request: Request):
+        super().__init__(request)
+        self._params = {
+            'action': 'parse',
+            'prop': 'text',
+            'page': '',
+            'origin': '*',
+            'format': 'json',
+            'formatversion': 2
+        }
+        self._data = ''
+        self._client_language = config.DEF_LANGUAGE
+        self.cors_config = {
+            "*": aiohttp_cors.ResourceOptions(
+                allow_credentials=True,
+                allow_headers='*',
+            )
+        }
+
+    async def get(self):
+        """Method process GET request, takes page from client and return json file"""
+        synopsis = {}
+
+        if 'pageName' in self.request.query:
+            self._params['page'] = self.request.query['pageName']
+
+        if 'language' in self.request.query:
+            self._client_language = self.request.query['language']
+
+        synopsis['text'] = await self.get_synopsis()
+        synopsis['text'] = self._content_parser()
+        synopsis['url'] = config.MEDIAWIKI_URL + self._params['page']
+        LOGGER.info('Response to the client`s request')
+
+        if synopsis['text'] == 'None':
+            return web.StreamResponse(status=404)
+        return web.json_response(synopsis)
+
+    async def get_synopsis(self):
+        """Method that parses the page content from mediawiki"""
+        self._params['page'] = f"{self._params['page']}/{self._client_language}"
+        content = await self._get_responce_mediawiki()
+
+        if content == '':
+            LOGGER.error('MediaWiki not found')
+            self._data = ''
+            return self._data
+
+        if 'error' in json.loads(content.text):
+            self._params['page'] = self._params['page'].replace(f'/{self._client_language}', '/en')
+            content = await self._get_responce_mediawiki()
+
+        try:
+            self._data = json.loads(content.text)
+            self._data = self._data['parse']['text']
+        except (AttributeError, TypeError, json.decoder.JSONDecodeError, KeyError):
+            self._data = ''
+            LOGGER.error('MediaWiki doesn`t return page text')
+        return self._data
+
+    async def _get_responce_mediawiki(self):
```

Comment:
1) There is a typo in the word `responce`.
2) Let's rename the method into `_request_mediawiki_api`.